app/renderer/js/utils/dnd-util.js

```diff
@@ -2,30 +2,168 @@
 
 const ConfigUtil = require(__dirname + '/config-util.js');
 
+const {ipcRenderer} = require('electron');
+
+let count = 0;
```

Comment:
What is the need of this variable?

Reply:
the count variable was linked to earlier commit in order to prevent the time selection menu on startup of the app, when the DND mode was already enabled. But, in successive commits, this prob was solved in other way, when i removed the copy of the checkDND and showDND from main.js. 
Now, its of no use. Thanks for pointing out. i will make all the recommended changes and push.